config/ide-helper.php

```diff
@@ -206,7 +206,7 @@
     | The value of the array is an array of type mappings. Key is the name of the custom type,
     | (for example, "jsonb" from Postgres 9.4) and the value is the name of the corresponding Doctrine2 type (in
     | our case it is 'json_array'. Doctrine types are listed here:
-    | http://doctrine-dbal.readthedocs.org/en/latest/reference/types.html
+    | https://www.doctrine-project.org/projects/doctrine-dbal/en/2.12/reference/types.html#types
```

Comment:
>  Pointed to 2.x as for the moment we end up with 2.x because of many other package dependencies.

But I think a link which won't get stale over time would be more preferred, what do you think?

```suggestion
    | https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/types.html#types
```

Reply:
I personally agree :)